static/js/ducks/allocations.js

```diff
@@ -9,9 +9,12 @@ const FETCH_ALLOCATIONS_OK = "skyportal/FETCH_ALLOCATIONS_OK";
 const REFRESH_ALLOCATIONS = "skyportal/REFRESH_ALLOCATIONS";
 
 // eslint-disable-next-line import/prefer-default-export
-export function fetchAllocations() {
-  return API.GET("/api/allocation", FETCH_ALLOCATIONS);
-}
+export const fetchAllocations = (params = {}) => {
+  if (!Object.keys(params).includes("apitype")) {
+    params.apitype = "api_classname";
+  }
```

Comment:
What happens if you call the API without specifying `apitype`? Is that not just handled gracefully by the handler's defaults? If so, do we need this here?

Reply:
I think we still want the API to return all allocations (both object and GCNEvent ones) if requested.

Reply:
Then why from the frontend we only want `api_classname`?

Reply:
deleted ;)